packet/flatcar-linux/kubernetes/calico/host-protection.yaml.tmpl

```diff
@@ -1,126 +1,109 @@
-# Customize inbound failsafe rules
-# https://docs.projectcalico.org/v3.5/getting-started/bare-metal/policy/failsafe
+# General
+# =======
+---
+# Disable default failsafe inbound rules
+apiVersion: crd.projectcalico.org/v1
+kind: FelixConfiguration
+metadata:
+  name: default
+spec:
+  failsafeInboundHostPorts: []
+
+# Host endpoints
+# ==============
+---
+${controller_host_endpoints}
+---
+${worker_host_endpoints}
+
+# Firewall policy
```

Comment:
I think it should be "Firewall policies"

Reply:
I prefer singular. "Policy" as in "this is the government's policy". We have one security policy which consists of multiple *rules*.

Reply:
Well, to be precise, we have multiple `GlobalNetworkPolicy` (so _policies_), each consist of one or more ingress and egress _rules_.

Reply:
I was referring to the "Lokomotive security policy", not the technical breakdown inside the source file. I am aware of how things are expressed in the file @invidian.

Reply:
That's very unclear reference then, could we change it to something more obvious then? :smile:

Reply:
Please suggest a new description. As said above, "Firewall policies" doesn't make sense to me, sorry.